great_expectations/render/renderer/microsoft_teams_renderer.py

```diff
@@ -20,196 +20,190 @@
 
 
 class MicrosoftTeamsRenderer(Renderer):
-    MICROSOFT_TEAMS_SCHEMA_URL = "http://adaptivecards.io/schemas/adaptive-card.json"
+    """
+    Responsible for formatting validation results and data docs links into a Microsoft Teams webhook
+    message payload.
+
+    Relevant links/documentation:
+        * Payload schema: https://adaptivecards.io/explorer/
+        * Interactive UI editor: https://adaptivecards.io/designer/
+
+    """
+
+    _MICROSOFT_TEAMS_CONTENT_TYPE = "application/vnd.microsoft.card.adaptive"
+    _MICROSOFT_TEAMS_SCHEMA_URL = "http://adaptivecards.io/schemas/adaptive-card.json"
+    _MICROSOFT_TEAMS_SCHEMA_VERSION = 1.5
+    _GX_LOGO_URL = "https://pbs.twimg.com/profile_images/1600908333493338112/GQQEPSKg_400x400.jpg"
```

Comment:
Picked a random GX image from google images but hoping to find a more official URL